lib/pages/chat/chat.dart

```diff
@@ -166,14 +169,7 @@ class ChatController extends State<Chat>
       );
     }
 
-    await showDialog(
```

Comment:
you must still keep dialog so user can preview before send

Reply:
out of scope